linux_service/moneo_service_deploy.sh

```diff
@@ -36,9 +35,9 @@ fi
 sudo chmod -R 777 $MONEO_PATH
 
 # Configure step
-echo "{
-    \"IDENTITY_CLIENT_ID\": \"$IDENTITY_CLIENT_ID\",
-    \"INGESTION_ENDPOINT\":  \"$INGESTION_ENDPOINT\" }" > $MONEO_PATH/src/worker/publisher/config/managed_prom_config.json
+jq '(.prom_config.IDENTITY_CLIENT_ID |= "'"$IDENTITY_CLIENT_ID"'")' "$MONEO_PATH/moneo_config.json" > "$MONEO_PATH/temp.json" && mv "$MONEO_PATH/temp.json" "$MONEO_PATH/moneo_config.json"
```

Comment:
Is jq being installed somewhere? Or is it assumed in the HPC image install?

Reply:
It is already installed as part of the HPC image. So, it assumes it is.